packs/sensu/etc/st2_handler.py

```diff
@@ -82,6 +81,9 @@ def _get_auth_token():
     except:
         raise Exception('Cannot get auth token from st2. Will try unauthed.')
```

Comment:
I had to switch this to sys.stderr.write() to get it working on a system with auth turned off.